auth/src/main/java/com/firebase/ui/auth/ui/FlowParameters.java

```diff
@@ -51,21 +51,36 @@
 
     public final boolean allowNewEmailAccounts;
 
+    public final boolean isReauth;
```

Comment:
Nevermind, I thought the `reauthenticate` method required an email but it doesn't so this won't work.